lib/grape/exceptions/base.rb

```diff
@@ -7,12 +7,12 @@ class Base < StandardError
       BASE_ATTRIBUTES_KEY = 'grape.errors.attributes'
       FALLBACK_LOCALE = :en
 
-      attr_reader :status, :message, :headers
```

Comment:
This is a breaking change like so, meaning that I cannot do .message on an exception, right? Let's reimplement `def message` that calls `to_s`?

Reply:
`message` exists through inheritance from StandardError

Reply:
I've just pushed a `spec` for `message`